plenum/server/primary_selector.py

```diff
@@ -246,6 +342,53 @@ def is_behind_for_view(self) -> bool:
                 return True
         return False
 
+    def _startCurrentStateSelection(self, view_no):
```

Comment:
Because it is a pair for _startSelection. Perhaps it is better to change to snake case